x-pack/docs/en/security/authentication/remote-clusters-privileges.asciidoc

```diff
@@ -33,8 +33,11 @@ On the remote cluster that contains the leader index, the {ccr} role requires
 the `read_ccr` cluster privilege, and `monitor` and `read` privileges on the
 leader index.
 
+NOTE: If requests will be authenticated with an <<security-api-create-api-key, API key>>, the API key
```

Comment:
```suggestion
NOTE: If requests are authenticated with an <<security-api-create-api-key, API key>>, the API key
```
Use present tense where possible.

Reply:
👍 I'll fix the other call out as well (the one about run-as)